config/routes.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root 'calculators#calculator'
```

Comment:
let's not set the `root`. later we may have some landing page, but for now, we have only a specific calculator page with URI like `calculators/diapers-calculator`

Reply:
Thanks, done.